pkg/validation/internal/testdata/objects/invalid_role_create_pdb.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: default
+  name: pdb-modifier
+rules:
+  - apiGroups: ["policy"]
+    resources: ["poddisruptionbudgets"]
+    verbs: ["create", "list"]
```

Comment:
nit: missing new line at the end of file.

Similar comments about the other yamls